src/System.Text.Json/tests/Utf8JsonReaderTests.TryGet.cs

```diff
@@ -7,6 +7,7 @@
 using System.Globalization;
 using System.IO;
 using System.Text.RegularExpressions;
+using Microsoft.SqlServer.Server;
```

Comment:
Why do we need this using directive?

Reply:
No clue. Git auto resolved it this way during a rebase.

Reply:
Fixed.